arches/app/templates/views/components/widgets/datepicker.htm

```diff
@@ -87,16 +89,19 @@
 </div>
 <div>
     <div class="form-group col-xs-12 pad-no crud-widget-container">
-        <div style="pad-hor">
+        <div class="pad-hor">
```

Comment:
becase style="pad-hor" didn't apply any valid style, we shouldn't make this into a class. We should just remove these references altogether.  Should just be a plain div.